includes/event/event-capabilities.php

```diff
@@ -5,18 +5,25 @@
 use Exception;
 use GP;
 use WP_User;
+use DateTimeImmutable;
+use DateTimeZone;
 use Wporg\TranslationEvents\Attendee\Attendee;
 use Wporg\TranslationEvents\Attendee\Attendee_Repository;
 use Wporg\TranslationEvents\Stats\Stats_Calculator;
 
 class Event_Capabilities {
-	private const MANAGE         = 'manage_translation_events';
-	private const CREATE         = 'create_translation_event';
-	private const VIEW           = 'view_translation_event';
-	private const EDIT           = 'edit_translation_event';
-	private const TRASH          = 'trash_translation_event';
-	private const DELETE         = 'delete_translation_event';
-	private const EDIT_ATTENDEES = 'edit_translation_event_attendees';
+	private const MANAGE           = 'manage_translation_events';
+	private const CREATE           = 'create_translation_event';
+	private const VIEW             = 'view_translation_event';
+	private const EDIT             = 'edit_translation_event';
+	private const TRASH            = 'trash_translation_event';
+	private const DELETE           = 'delete_translation_event';
+	private const EDIT_ATTENDEES   = 'edit_translation_event_attendees';
```

Comment:
(Commenting here since I can't comment on `has_edit_attendees()` because it hasn't been modified)

I'm wondering, should we restrict editing attendees after the event has ended, similarly to what we're now doing for other fields? Currently, editing attendees is allowed after the event ends.

Reply:
I agree but we would need the attendees page to be accessible after the event ends. We would have to find a way to make the attendees page accessible if we add the logic to check for event end date in this capability.